2015-06-11 11:09:25,033 INFO:Demo03.py:8 Timer started 

2015-06-11 11:09:25,033 DEBUG:PyroUtil.py:63 Can connect to a LISTENING port of nameserver on 147.32.130.137:9090 

2015-06-11 11:09:25,035 DEBUG:PyroUtil.py:74 Connected to NameServer on 147.32.130.137:9090. Pyro4 version on this computer is 4.35 

2015-06-11 11:09:25,035 DEBUG:PyroUtil.py:205 Creating ssh tunnel via command: ssh -L 5555:147.32.130.150:44361 mmp@147.32.130.150 -N -oStrictHostKeyChecking=no 

2015-06-11 11:09:26,039 DEBUG:PyroUtil.py:95 Found URI PYRO:obj_197b32c9af2f48c7bf3459cdf3e9fe3a@127.0.0.1:5555 from a nameServer <Pyro4.core.Proxy at 0x7f16ce0e7910, connected, for PYRO:Pyro.NameServer@147.32.130.137:9090> 

2015-06-11 11:09:26,041 DEBUG:PyroUtil.py:104 Cannot connect to application Mupif.JobManager@demo. Is the server running? 

2015-06-11 11:09:26,042 ERROR:PyroUtil.py:105 receiving: not enough data 
Traceback (most recent call last):
  File "../../mupif/PyroUtil.py", line 101, in connectApp
    sig = app2.getApplicationSignature()
  File "/usr/local/lib/python2.7/dist-packages/Pyro4/core.py", line 230, in __getattr__
    self._pyroGetMetadata()
  File "/usr/local/lib/python2.7/dist-packages/Pyro4/core.py", line 483, in _pyroGetMetadata
    self.__pyroCreateConnection()
  File "/usr/local/lib/python2.7/dist-packages/Pyro4/core.py", line 435, in __pyroCreateConnection
    msg = message.Message.recv(conn, None, hmac_key=self._pyroHmacKey)
  File "/usr/local/lib/python2.7/dist-packages/Pyro4/message.py", line 161, in recv
    msg = cls.from_header(connection.recv(cls.header_size))
  File "/usr/local/lib/python2.7/dist-packages/Pyro4/socketutil.py", line 447, in recv
    return receiveData(self.sock, size)
  File "/usr/local/lib/python2.7/dist-packages/Pyro4/socketutil.py", line 182, in receiveData
    raise err
ConnectionClosedError: receiving: not enough data
2015-06-11 11:09:26,044 INFO:PyroUtil.py:291 jobMan.allocateJob() failed 

2015-06-11 11:09:26,044 ERROR:PyroUtil.py:292 'NoneType' object has no attribute 'allocateJob' 
Traceback (most recent call last):
  File "../../mupif/PyroUtil.py", line 288, in allocateApplicationWithJobManager
    retRec = jobMan.allocateJob(getUserInfo(), natPort=natPort)
AttributeError: 'NoneType' object has no attribute 'allocateJob'
2015-06-11 11:09:26,044 ERROR:Demo03.py:19 'NoneType' object has no attribute 'allocateJob' 
Traceback (most recent call last):
  File "Demo03.py", line 16, in <module>
    appRec = PyroUtil.allocateApplicationWithJobManager( ns, cConf.demoJobManRec, cConf.jobNatPorts.pop(0), cConf.sshClient, cConf.options, cConf.sshHost )
  File "../../mupif/PyroUtil.py", line 293, in allocateApplicationWithJobManager
    raise (e)
AttributeError: 'NoneType' object has no attribute 'allocateJob'
